src/pages/projects/edit/Edit.tsx

```diff
@@ -82,7 +82,6 @@ export default function Edit() {
           value={project?.client_id}
           onChange={(id) => handleChange('client_id', id)}
           readonly
-          staleTime={Infinity}
```

Comment:
@beganovich @turbo124 Setting staleTime to Infinity manually for the Edit Project page is no longer necessary because we have already changed it to be set by default.